### PR TITLE
[MT-99]:(feature) EAS update configuration

### PR DESCRIPTION
**Pull request by PixoDev**
Allow OTA updates using EAS services with expo updates

https://docs.expo.dev/eas-update/introduction/

### CODE REVIEW

File: ios/Internxt.xcodeproj/xcshareddata/xcschemes/Internxt.xcscheme [hunk 1]
```diff
@@ -41,7 +41,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (sg-gs) [inline ios/Internxt.xcodeproj/xcshareddata/xcschemes/Internxt.xcscheme:44]: is this intentional?

File: ios/Internxt/AppDelegate.mm [hunk 2]
```diff
@@ -0,0 +1,166 @@
+#import "AppDelegate.h"
```
Comment (sg-gs) [inline ios/Internxt/AppDelegate.mm:1]: Please add a link reference to this source code or documentation if its own-made if possible

**Review by sg-gs — APPROVED** [review]

File: ios/Internxt/AppDelegate.mm (hunk 2 above)
Comment (PixoDev) [reply ios/Internxt/AppDelegate.mm:1]: this code is boilerplate only

File: ios/Internxt.xcodeproj/xcshareddata/xcschemes/Internxt.xcscheme (hunk 1 above)
Comment (PixoDev) [reply ios/Internxt.xcodeproj/xcshareddata/xcschemes/Internxt.xcscheme:44]: those changes are auto generated from `expo prebuild` command, is not intentional but is not something we can prevent

**Pull request merged**